src/components/Form/MucCheckbox.stories.ts

```diff
@@ -0,0 +1,23 @@
+import MucCheckbox from "./MucCheckbox.vue";
+
+export default {
+  component: MucCheckbox,
+  title: "Input/MucCheckBox",
```

Comment:
```suggestion
  title: "Forms/MucCheckBox",
```

I would prefer a "Forms"-Category